projects/github-actions/repo-gardening/src/tasks/gather-support-references/index.js

```diff
@@ -44,7 +50,7 @@ async function getListComment( issueComments ) {
  */
 async function getIssueReferences( octokit, owner, repo, number, issueComments ) {
 	const ticketReferences = [];
-	const referencesRegexP = /[0-9]*-(?:zen|zd)/gim;
+	const referencesRegexP = /([0-9]*-(?:zen|zd)|[a-zA-Z0-9-]+-p2#comment-[0-9]*)/gim;
```

Comment:
Is it intentional that we're only doing P2 comments, and not references to entire P2 posts?

Also, I don't think the extra parens are needed, this should work
```suggestion
	const referencesRegexP = /[0-9]*-(?:zen|zd)|[a-zA-Z0-9-]+-p2#comment-[0-9]*/gim;
```

Reply:
> Is it intentional that we're only doing P2 comments, and not references to entire P2 posts?

Yes, I chose to do that since we often use references to p2 posts in issues but they're not always support references. Comments on p2 posts, when we refer to them in issues, can almost always be considered as support references though, in my experience.